docs/extend/extend-apiml/api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer.md

```diff
@@ -266,13 +270,11 @@ After you add API Mediation Layer integration endpoints, you are ready to add se
     routedServices:
     - gatewayUrl: api/v1
       serviceUrl: /hellospring/api/v1
-    
```

Comment:
In this case we need api-doc endpoint to access swagger. Return back this part.

Reply:
Done